### ISSUE TITLE
Config reload failed with "No client connected to db_id 'CONFIG_DB'"

### ISSUE THREAD

**Issue by wangxin** [Azure/sonic-buildimage#3817]
**Description**
Run "config reload -y" always failed with below error on latest master image.

```
$ sudo config reload -y
Stopping service swss ...
Stopping service lldp ...
Stopping service bgp ...
Stopping service hostcfgd ...
Traceback (most recent call last):
  File "/usr/bin/config", line 12, in <module>
    sys.exit(config())
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/config/main.py", line 479, in reload
    client = config_db.redis_clients[config_db.CONFIG_DB]
  File "/usr/local/lib/python2.7/dist-packages/swsssdk/interface.py", line 78, in __getitem__
    raise MissingClientError("No client connected for db_id '{}'".format(item))
swsssdk.exceptions.MissingClientError: No client connected for db_id 'CONFIG_DB'
```

Tested image: SONiC.HEAD.132-45e13b99

The switch is also started without PortChannel IP address:
```
$ show ip interface
Interface    IPv4 address/mask    Admin/Oper    BGP Neighbor    Neighbor IP
-----------  -------------------  ------------  --------------  -------------
Loopback0    10.1.0.32/32         up/up         N/A             N/A
docker0      240.127.1.1/24       up/down       N/A             N/A
eth0         10.210.25.44/22      up/up         N/A             N/A
lo           127.0.0.1/8          up/up         N/A             N/A

```

**Steps to reproduce the issue:**
1. The switch is running t0 topology.
2. Install the latest SONiC image built from master branch to switch.
3. Check interface IP address using "show ip interface"
4. Perform config reload using "sudo config reload -y".

**Describe the results you received:**
1. PortChannels have no IP address. 
2. No BGP session is UP.
3. Config reload always failed with "No client connected to db_id 'CONFIG_DB'"

**Describe the results you expected:**
1. The PortChannel should have IP address up.
2. BGP sessions should be up.
3. Config reload should not fail.

**Additional information you deem important (e.g. issue happens only occasionally):**

    **Output of `show version`:**

```
$ show version

SONiC Software Version: SONiC.HEAD.132-45e13b99
Distribution: Debian 9.11
Kernel: 4.9.0-9-2-amd64
Build commit: 45e13b99
Build date: Sun Nov 24 08:10:31 UTC 2019
Built by: johnar@jenkins-worker-4

Platform: x86_64-mlnx_msn2100-r0
HwSKU: ACS-MSN2100
ASIC: mellanox
Serial Number: MT1851X13655
Uptime: 14:28:27 up 15 min,  2 users,  load average: 0.05, 1.49, 1.61

Docker images:
REPOSITORY                 TAG                 IMAGE ID            SIZE
docker-syncd-mlnx          HEAD.132-45e13b99   68886e835f08        373MB
docker-syncd-mlnx          latest              68886e835f08        373MB
docker-fpm-frr             HEAD.132-45e13b99   90ddd4bc3b6b        321MB
docker-fpm-frr             latest              90ddd4bc3b6b        321MB
docker-sflow               HEAD.132-45e13b99   79a57f618cd8        305MB
docker-sflow               latest              79a57f618cd8        305MB
docker-lldp-sv2            HEAD.132-45e13b99   32b91649593c        299MB
docker-lldp-sv2            latest              32b91649593c        299MB
docker-dhcp-relay          HEAD.132-45e13b99   caac214e5f80        289MB
docker-dhcp-relay          latest              caac214e5f80        289MB
docker-database            HEAD.132-45e13b99   e5525001be5a        281MB
docker-database            latest              e5525001be5a        281MB
docker-snmp-sv2            HEAD.132-45e13b99   0dd0b989b087        335MB
docker-snmp-sv2            latest              0dd0b989b087        335MB
docker-orchagent           HEAD.132-45e13b99   00dff77bd0d8        322MB
docker-orchagent           latest              00dff77bd0d8        322MB
docker-teamd               HEAD.132-45e13b99   d2ce993e4862        304MB
docker-teamd               latest              d2ce993e4862        304MB
docker-sonic-telemetry     HEAD.132-45e13b99   d6e5242f0e71        304MB
docker-sonic-telemetry     latest              d6e5242f0e71        304MB
docker-router-advertiser   HEAD.132-45e13b99   5723ee008d65        281MB
docker-router-advertiser   latest              5723ee008d65        281MB
docker-platform-monitor    HEAD.132-45e13b99   110fed0921d2        565MB
docker-platform-monitor    latest              110fed0921d2        565MB
```

    **Attach debug file `sudo generate_dump`:**

[sonic_dump_arc-switch1029_20191125_143744.zip](https://github.com/Azure/sonic-buildimage/files/3887120/sonic_dump_arc-switch1029_20191125_143744.zip)

**Comment by dzhangalibaba** [comment]
Now Guohan revert the sonic-py-swss submodule ref update, now there should be  no issue.

The issue happened because we update sonic-py-swsssdk lib which has some API changes, all the application should use the API to get data instead of accessing internal data directly. 

While the we missed to find out the script in sonic-utilities which is still accessing internal data directly. We should change sonic-utilities first and then update sonic-py-swsssdk submodule.

One PR is in sonic-py-swsssdk which we already merged  but the ref point is not updated yet.
https://github.com/Azure/sonic-py-swsssdk/pull/52

The other one is in sonic-utilities which I just submitted today.  This part needs to be merged first and then we can update the sonic-py-swsssdk submodule ref. 
https://github.com/Azure/sonic-utilities/pull/753

I did a global search, these two PR should cover all invalid cases which accessing internal data db_map[] and redis_clients[].
